tools/migration/main_test.go

```diff
@@ -0,0 +1,78 @@
+package main_test
+
+import (
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
+	ast "github.com/stretchr/testify/assert"
+	req "github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/testhelpers"
+)
+
+func TestUsage(t *testing.T) {
+	tf.UnitTest(t)
```

Comment:
If these tests are calling `exec.Command` we should label them as `Integration` tests, there is an (undocumented..sorry) expectation that unit tests don't require anything to be built when running.